lib/duffel_api/errors/error.rb

```diff
@@ -75,9 +75,7 @@ def code
       # Returns the request ID of the request that generated the error.
       #
       # @return [String]
-      def request_id
-        api_response.request_id
-      end
+      delegate :request_id, to: :api_response
```

Comment:
@timrogers The `activesupport` gem provides this method. It's not built-into Ruby itself.

Reply:
So presumptuous!